Cargo.toml

```diff
@@ -9,4 +9,8 @@ members = [
     "proptest-derive",
 ]
 
-exclude = ["proptest/test-persistence-location/*"]
+exclude = [
+    "proptest/test-persistence-location/*",
+    "proptest/examples",
+    "kani",
```

Comment:
Remove this once we publish the kani crate

Reply:
#1639